packages/musl_xz.rb

```diff
@@ -6,7 +6,7 @@ class Musl_xz < Package
   version '5.2.5-e7da-1'
   license 'public-domain, LGPL-2.1+ and GPL-2+'
   compatibility 'all'
-  source_url 'https://github.com/xz-mirror/xz.git'
+  source_url 'https://github.com/tukaani-project/xz.git'
```

Comment:
It might be better to keep the previous source url.  A mirror site will localize the download.

Reply:
The previous url is archived and tells you to go to the other url, so I switched.
![image](https://github.com/chromebrew/chromebrew/assets/64618338/8dfc50c0-54e9-47ee-9b7f-d219459df5d2)

Reply:
Fair enough.